stratum/hal/lib/common/hal.cc

```diff
@@ -4,6 +4,9 @@
 
 #include "stratum/hal/lib/common/hal.h"
 
+#include <limits.h>
```

Comment:
Do we need limits.h anymore?
```suggestion
```

Reply:
```
> grep -nr "PIPE_BUF" /usr/include/
/usr/include/linux/limits.h:14:#define PIPE_BUF        4096	/* # bytes in atomic write to a pipe */
```

Reply:
I think this is needed for PIPE_BUF, nevermind...

Reply:
We could remove the `static_assert`, though.

Reply:
I guess it doesn't hurt, but it also seems a bit unnecessary. The value in the docs is >> what we use, and it's been that way for decades.